website/activemembers/apps.py

```diff
@@ -13,3 +13,28 @@ def ready(self):
         """Import the signals when the app is ready."""
         # pylint: disable=unused-import,import-outside-toplevel
         from . import signals
+
+    def menu_items(self):
+        return {
+            "categories": [{"name": "association", "title": "Association", "key": 1}],
+            "items": [
+                {
+                    "category": "association",
+                    "title": "Board",
+                    "viewname": "activemembers:boards",
```

Comment:
For one, its not view name but url name. 
Also, I think it's better to already include the URL itself in here with a reverse or reverse_lazy (if that's possible, at least)

Reply:
Django calls this viewname themselves: https://docs.djangoproject.com/en/4.1/ref/urlresolvers/#django.urls.reverse

I think reversing it beforehand technically removes some flexibility in the menu creating. I think it would be fine though

Reply:
> Django calls this viewname themselves: https://docs.djangoproject.com/en/4.1/ref/urlresolvers/#django.urls.reverse


Weird haha, but sure 


> I think reversing it beforehand technically removes some flexibility in the menu creating. I think it would be fine though

If I would want to add a URL to the menu to an external page, that's currently not possible without creating a redirect view for it.

Reply:
> If I would want to add a URL to the menu to an external page, that's currently not possible without creating a redirect view for it.

https://github.com/svthalia/concrexit/blob/e98a4809442e92762e3315a6191576f434343a33/website/members/apps.py#L26-L32

Reply:
Another solution would just be to add `"noreferer": True` to the item. I personally think that that would be the cleanest solution.